setup.py

```diff
@@ -53,6 +53,7 @@
     install_requires=[
         'pyxdg',
         'requests',
+        'libarchive-c',
```

Comment:
I don't really understand which deps should be in install_requires and which should be in the requirements file.